src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -597,28 +627,28 @@ <h2 class="m1">Apple</h2>
         </div>
       </form>
 
-    <!-- ## Collapsibles / Accordion -->
-    <!-- Use `amp-accordion` to hide and show additional data about your product. Learn more about `amp-accordion` [here](/components/amp-accordion/). -->
-    <div>
-      <amp-accordion>
-        <section>
-          <h4 class="p1">Description</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Specification</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Measurements</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-        <section>
-          <h4 class="p1">Other</h4>
-          <p class="p1">Lorem ipsum dolor sit amet, mauris id, fermentum natoque ut et ullam dolore egestas. Vitae tortor maecenas nec consequat, nisl vestibulum est est curabitur. Mi eu in pellentesque lectus nostra ut. </p>
-        </section>
-      </amp-accordion>
-    </div>
+    <!-- ## Tab panels -->
+    <!-- Use `amp-selector` styled as tab panels to add additional data about your product. Learn more about [`amp-selector`](https://www.ampproject.org/docs/reference/components/dynamic/amp-selector). -->
```

Comment:
Link to our tab panel sample instead. `Learn how to implement tabs in AMP [here](...).